src/rust/bitbox02-rust/src/hww/api/cardano/sign_transaction/certificates.rs

```diff
@@ -88,6 +88,25 @@ pub async fn verify<'a>(
                 })
                 .await?;
             }
+            VoteDelegation(certificate::VoteDelegation {
+                keypath,
+                r#type: _,
+                drep_credhash: _,
+            }) => {
+                validate_address_shelley_stake(keypath, Some(bip44_account))?;
+                signing_keypaths.push(keypath);
+                confirm::confirm(&confirm::Params {
```

Comment:
Don't we also have to confirm the drep type and drep credhash?

Reply:
Added a second confirm screen to do this, I didn't want to merge into one because it made it a bit unreadable in my opinion.